src/vcCamera.cpp

```diff
@@ -238,11 +238,11 @@ void vcCamera_Apply(vcState *pProgramState, vcCamera *pCamera, vcCameraSettings
     // Translation
     if (pCamSettings->cameraMode == vcCM_FreeRoam)
     {
-      if (pCamSettings->moveMode == vcCMM_Plane)
+      if (!pCamSettings->lockAltitude)
```

Comment:
Swap cases around to avoid negative test? I appreciate it's the simple case but `if not condition else` is a weird pattern.

Reply:
Fixed in a future PR. Already made the change locally so it isn't missed.